### PR TITLE
Remove excessive anchor links [merge on approval]

### PR DESCRIPTION
**Pull request by ry**

### CODE REVIEW

File: website/index.html [hunk 1]
```diff
@@ -101,14 +101,14 @@ <h1 id="deno"><a href="#deno">#</a>Deno</h1>
...
-      <p id="install-shell"><a href="#install-shell">#</a>With Shell</p>
+      <p id="install-shell">With Shell</p>
       <pre>curl -fL https://deno.land/x/install/install.sh | sh</pre>
-      <p id="install-powershell"><a href="#install-powershell">#</a>With PowerShell</p>
+      <p id="install-powershell">With PowerShell</p>
       <pre>iex (iwr https://deno.land/x/install/install.ps1)</pre>
 
       <h2 id="mini-tutorial"><a href="#mini-tutorial">#</a>Mini-tutorial</h2>
 
-      <p id="try"><a href="#try">#</a>Try a Deno program. This one serves a local directory in HTTP.</p>
```
Comment (sh7dm) [inline website/index.html]: I think this is useful if we want to link usage example

File: website/index.html [hunk 2]
```diff
@@ -101,14 +101,14 @@ <h1 id="deno"><a href="#deno">#</a>Deno</h1>
 
       <h2 id="install"><a href="#install">#</a>Install</h2>
 
-      <p id="install-shell"><a href="#install-shell">#</a>With Shell</p>
+      <p id="install-shell">With Shell</p>
       <pre>curl -fL https://deno.land/x/install/install.sh | sh</pre>
-      <p id="install-powershell"><a href="#install-powershell">#</a>With PowerShell</p>
```
Comment (sh7dm) [inline website/index.html]: If you remove link, remove the `id` too, it isn't needed anymore

File: website/index.html (hunk 1 above)
Comment (ry) [reply website/index.html]: There's an anchor directly above this in the title of the section...

File: website/index.html (hunk 2 above)
Comment (ry) [reply website/index.html]: Fixed.

File: website/index.html (hunk 1 above)
Comment (sh7dm) [reply website/index.html]: Okay

**Pull request merged**